### DISCUSSION TITLE
How to rank the strength of the results

### DISCUSSION THREAD

**Discussion by slucey** [NOAA-EDAB/RpathQNM#7] [Ideas]
So I'm struggling a bit with assigning a strength (low, medium, high) to the results because there are three categories.  

My initial thought is to score them as a sum of dominant direction plus half of the neutral outcomes and rank as:

| Score     | Strength |
| ----------- | ----------- |
| 600 - 750  | Low       |
| 751 - 900  | Med       |
| 901 - 1000 | High |
 
For example:

| Negative | Neutral | Positive | Score | Strength |
| ---------- | -------- | --------- | ------ | -------- |
| 1000 | 0 | 0 | 1000 | High |
| 800 | 200 | 0 | 900 | High |
| 800 | 0 | 200 | 800 | Med |
| 750 | 120 | 130 |  810 | Med |
| 600 | 400 | 0 | 800 | Med |
| 600 | 200 | 200 | 700 | Low |
| 600 | 0 | 400 | 600 | Low |

So the top two would get the biggest negative symbol, the next 3 a medium sized symbol, and the last two the small symbol.  

I'm open to suggestions...especially regarding the ranges.

**Comment by sgaichas** [comment]
That does get challenging. I'm wondering if we lose any information by ignoring the other categories when assigning strength? So simplify a bit, and rank only based on the dominant category.

We could then apply the scores for strength as above but without adding in the neutrals.

Also, we could go back and see how often we had the other two categories roughly equal (indicating more uncertainty about the outcome?) and how often it was only one other category?

**Comment by jamiectam** [comment]
I was doing something similar to what Sean suggests, but probably much less fancy. I'm fine with the ranges.  So then whatever has ~500positive and ~500negative would be catagorized as uncertain/mixed?

**Reply by slucey** [reply]
Yes...any scenario that does not have one of the three categories >= 600 is classified as mixed.
